src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -248,9 +250,29 @@ private void setupKeyboardShortcuts() {
                     break;
                 }
             }
+            if (SHOW_RELATED_ISSUE_OR_PR.match(event) && ui.getBrowserComponent().isCurrentUrlIssue()) {
+                if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
+                    showRelatedIssueOrPR();
+                } else { // only for default. can remove if default key for MARK_AS_READ_CHANGES
+                    markAsRead();
+                }
+            }
         });
     }
 
+    private void showRelatedIssueOrPR() {
+        TurboIssue issue = listView.getSelectedItem().get();
```

Comment:
You have to check here if there is a selected item. This would be like an NPE otherwise.